src/components/User/Limits/Limits.Reviewer.vue

```diff
@@ -483,11 +493,21 @@ export default {
       this.rejectForm.isShown = false
     },
 
+    isDatalistFormValid () {
+      const normalizedDocsList = this.getNormalizedDocsList()
+      this.uploadDocs.forEach(
+        val => { val.isDocValid = normalizedDocsList.includes(val.label) })
```

Comment:
val => item

Reply:
isDocValid - the name of this variable implies that it have Boolean type of value